README.md

```diff
@@ -2,7 +2,10 @@
 
```

Comment:
# Autoscale Core Library

Adding 'library' to the title should help people notice that this is a library

Reply:
added.